engine/flink/components/dev-table/src/main/scala/pl/touk/nussknacker/engine/flink/table/extractor/SqlExtractorErrors.scala

```diff
@@ -0,0 +1,50 @@
+package pl.touk.nussknacker.engine.flink.table.extractor
+
+import pl.touk.nussknacker.engine.flink.table.extractor.SqlStatementReader.SqlStatement
+
+final case class SqlExtractorError(
+    errorType: SqlExtractionErrorType,
```

Comment:
Neight of this fields is used. YAGNI.

Reply:
They were supposed to be logged/part of exception message - I've added a message val based on these fields that is part of exception now